var/spack/repos/builtin/packages/py-databricks-cli/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyDatabricksCli(PythonPackage):
+    """A command line interface for Databricks."""
+
+    homepage = "https://pypi.org/project/databricks-cli/"
+    pypi = 'databricks-cli/databricks_cli-0.17.4.tar.gz'
```

Comment:
Looks like source distributions use dashes, not underscores.  At least that is the case for 0.17.4.

Reply:
Making that correction, I *can* confirm the sha256 for `0.17.4` and `0.14.3`.